docker/almalinux-image.docker

```diff
@@ -47,6 +47,7 @@ RUN mkdir -p $RPM_BUILD_DIR/{BUILD,BUILDROOT,RPMS,SOURCES,SPECS,SRPMS}
 
 # Clone Cassandra and cache maven artifacts
 ARG CASSANDRA_GIT_URL=https://github.com/apache/cassandra.git
+RUN git config --global http.postBuffer 524288000
```

Comment:
on a number of recent failures this has been needed.

Reply:
Only for this image?